Makefile

```diff
@@ -1,14 +1,17 @@
 BINDIR ?= /usr/bin
 SBINDIR ?= /usr/sbin
 MANDIR ?= /usr/share/man/man1
+JOBS = $(shell getconf _NPROCESSORS_ONLN)
```

Comment:
I checked, and dune is pretty good at doing this on its own now since version 1.0.0: https://github.com/ocaml/dune/blob/master/CHANGES.md#100-10072018
So I think we can drop the -j and JOBS flag now that we switched to `dune`.